docs/getting_started/hopsworksai/guides/getting_started.rst

```diff
@@ -219,9 +219,8 @@ The instance will start. This might take a couple of minutes:
     :align: center
     :figclass: align-center
 
-As soon as the instance has started, you will be able to log in to your new Hopsworks instance with the username
-and password provided. We recommend that you change that password after your first login.
-You are also able to stop or terminate the instance.
+As soon as the cluster has started, you will be able to log in to your new Hopsworks cluster with the username
+and password provided. You are also able to stop or terminate the cluster.
```

Comment:
Time concordance, one centence say `you will` and the next says `you are`.

Reply:
done